api/types/constants.go

```diff
@@ -783,6 +783,10 @@ const (
 	DiscoveryLabelLDAPPrefix = "ldap/"
 )
 
+// CloudLabelPrefixes are prefixes used by cloud labels, generally added when
+// using automatic discovery
+var CloudLabelPrefixes = []string{CloudAWS, CloudAzure, CloudGCP}
```

Comment:
You could include `DiscoveryLabelLDAPPrefix` in here too. Not exactly "cloud" - but the same idea (labels synchronized from some other source that get populated automatically)